wdl2cwl/main.py

```diff
@@ -643,6 +656,12 @@ def convert(workflow: str) -> str:
             )
         )
 
+    if "disks" in ast.task_runtime:
+        outdir_min=get_outdir_min(ast.task_runtime["disks"])
+    else:
+        outdir_min=get_outdir_min("1")
+    requirements.append(cwl.ResourceRequirement(outdirMin=outdir_min)
```

Comment:
```suggestion
    outdir_min=get_outdir_min(getattr(ast.task_runtime, "disks", "1"))
    requirements.append(cwl.ResourceRequirement(outdirMin=outdir_min))
```
Even further simplify